packages/app/src/debug/DebugFailedTest.vue

```diff
@@ -121,6 +133,7 @@ const failedTestData = computed(() => {
         {
           title: '...',
           type: 'ELLIPSIS',
+          originalTitle: ele,
```

Comment:
This will need to gather up all the "MIDDLE" parts to construct the `originalTitle` passed to the ELLIPSIS tooltip

Reply:
That makes sense, thanks! I think I was missing the underlying concept here, that it's really combining the parts 👍